build.gradle

```diff
@@ -9,11 +9,12 @@ repositories {
 
 dependencies {
 	compile "com.google.code.gson:gson:2.3"
-
+	compile "org.apache.commons:commons-lang3:3.0"
```

Comment:
Removed dependency from the project and replaced occurrence of `isNotBlank` with `not null + not empty` check.